public/app/features/datasources/components/picker/DataSourceDropdown.tsx

```diff
@@ -406,7 +407,7 @@ function Footer({ onClose, onChange, onClickAddCSV, ...props }: FooterProps) {
             ref={props.footerRef}
             onKeyDown={isUploadFileEnabled ? onKeyDownFirstButton : onKeyDownLastButton}
           >
-            Open advanced data source picker
+            <Trans i18nKey="data-sources.advanced-datasource-picker">Open advanced data source picker</Trans>
```

Comment:
@grafana/grafana-frontend-platform: 

- Any guidelines on naming the key for this one? It's a generic component, should it be called `data-sources.advanced-datasource-picker` or more something like `data-source-picker.advanced-datasource-picker`.
- Also I see some translation keys use `data-source`, others `datasource`. Which one is preferred? Or maybe it doesn't matter?

Reply:
Thank you for your feedback I appreciate it.
- we can name it to `data-source-picker.advanced-datasource-picker`. 
- I used `data-source` at the top level and `datasource` at the nested levels. We can use `data-source` at all levels for consistency.

Thanks again, I have addressed both feedbacks.

Reply:
Here we should keep it in `data-sources` as the first part of the key, because we already have the key `data-sources.datasource-add-button.label` for the `DataSourceAddButton` which is part of the same folder structure. I would go with something like `data-sources.picker.open-advanced-button`, wdyt?

Reply:
I thought the first level of the key was related to the page the component is shown. In the case of the existing "add button" - it belongs to "Data sources" config/admin page. However, that admin page does not really contain a picker anywhere (the picker is more a generic component used in other places). Should we still use `data-sources.` namespace for the picker?

Reply:
Sorry missed the comments last week! Ah if it is generic then I agree let's call it something like you mentioned. Maybe `data-source-picker.open-advanced-button`? Just to make it clear it's the text inside a button